connectors/v2/ego-fm.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#playerInfo';
+
+Connector.artistSelector = '#current > span.artist';
+
+Connector.trackSelector = '#current > span.song';
+
+Connector.isPlaying = function () {
```

Comment:
Unfortunately clicking the play button opens a new window, but does not change anything else in the page...